comfy_cli/command/generate/client.py

```
@@ -0,0 +1,148 @@
+            if not path.is_file():
+                raise ApiError(0, "", f"--{name}: file not found: {path}")
+            files.append((name, (path.name, path.open("rb"), "application/octet-stream")))
+        elif flag and flag.kind == "array" and flag.item_kind == "binary":
+            for p in value:
+                p = Path(p) if not isinstance(p, Path) else p
+                if not p.is_file():
+                    raise ApiError(0, "", f"--{name}: file not found: {p}")
+                files.append((name, (p.name, p.open("rb"), "application/octet-stream")))
+        elif flag and flag.kind in ("object", "array"):
+            # Multipart form fields are scalar — JSON-encode complex values.
+            import json as _json
```

Pylint (4.0.5)

[convention] Import outside toplevel (json)

(C0415)